README.md

```diff
@@ -1,11 +1,66 @@
-# jupiter-repo-dependencies
+# JupiterOne Repository Relationship Generator
 
-- Put the repos you want to run the script on in the `repos` folder.
-- Use `yarn deps` to run the script.
+A script that generates graphql relationships between a repository and its dependencies.
+This script will look for dependencies within a repository's `package.json` file as well as the `dependencies.yaml`file if the repository has a `deploy` directory.
 
-## Package Scope Input
+This script uses the JupiterOne graphql graph database.
 
-- You will be asked to input as many package scopes as you like.
-- For example, if you have packages that follow this pattern: @package/*, then you would input @package.
+## Before Running
+- Create a directory containing the repositories that you would like the script to be run on anywhere you would like in the project (Note: You will need the path to this directory while running the script).
 
+- Ensure that the `package.json` for each repository is at the root of the repository.
+
+- JupiterOne Client credentials are required to run the necessary queries to create the relationships.
+  - There are 4 parameters that need to be passed in:
+    - Account
+    - Username
+    - Password
+    - AccessToken
```

Comment:
I've never looked closely, does the API require username/password AND an access token? Coming at this with a bit of inexperience, it seems surprising that I would have to give my username/password at all. Perhaps there could be a link to the JupiterOne API authentication documentation.